src/Logger.cpp

```diff
@@ -27,7 +27,7 @@ std::unique_ptr<ASTConsumer> LoggerFrontendAction::CreateASTConsumer(CompilerIns
 void LoggerFrontendAction::EndSourceFileAction() {
     auto current_file = this->getCurrentFile().str();
     auto current_file_path = fs::path(current_file);
-    auto log_path = current_file_path.parent_path().parent_path() / fs::path("log.txt");
+    auto log_path = current_file_path.parent_path().parent_path() / fs::path("log_1.txt");
```

Comment:
Maybe it should be an argument in the config? I don't like this `_1` in the name...

Reply:
Seems like a error, it should be `log_{transform_index + 1}.txt`